website/integrations/services/slack/index.md

```diff
@@ -0,0 +1,75 @@
+Your Slack integration requires two property mappings, one each for `User.Email` and `User.Username`, so that authentik can retrieve and map these values from Slack.
+
+1. Log in as admin to your authentik instance and then click **Admin interface**.
+2. Navigate to **Customization -> Property Mappings**.
+3. Create the property mapping for `User.Email`.
+    1. On the **Property Mappings** page, click **Create**.
+    2. On the **New property mapping** modal, select **SAML Property Mapping** and then click **Next**.
+    3. Define the required values. In the **Expression** field, define `User.Email` as `return request.user.email`.
+4. Click **Finish**.
+5. Create the property mapping for `User.Username`.
+    1. On the **Property Mappings** page, click **Create**.
+    2. On the **New property mapping** modal, select **SAML Property Mapping** and then click **Next**.
+    3. Define the required values. In the **Expression** field, define `User.Username` as `return request.user.username`.
+6. Click **Finish**.
+
+### Step 2. Create a new authentication provider
+
+1. Navigate to **Applications -> Providers** and then click **Create**.
+2. On the **New provider** modal, select **SAML Provider** and then click **Next**.
+3. Define the following values (values not listed below can be left as default or empty):
+    - **Name**: provide a clear name, such as "slack".
+    - **Authorization flow**: Authorize Application (`default-provider-authorization-implicit-consent`).
+    - **Protocol settings** define the following values:
+        - **ACS URL**: `https:_workspace-name_.slack.com/sso/saml`
+        - **Issuer**: `https://slack.com`.
+        - **Service Provider Binding**: select **Post**
+    - **Advanced protocol settings**
+        - **Signing Certificate**: select the appproriate certificate for Slack.
+        - **Property mappings**: Select the property mappings that you created in Step 1. You can leave the default property mappings and other settings.
+4. Click **Finish** to create the provider.
+
+### Step 3. Create a new application
+
+1. Navigate to **Applications -> Applications** and then click **Create**.
+2. Provide a name for the new application.
+3. Set the provider to the one you just created.
+4. Ensure that the **Policy engine mode** is set to **ANY, any policy must match to grant access**.
+5. Click **Create**.
+
+:::info
+After you have created the provider and application, and the application is connected to the provider (Step 3 above) the **Overview** tab on the provider's detail page in the Admin UI will display additional information that you will need to configure Slack, using the following steps.
+:::
+
+## Slack configuration
+
+### Step 4. Configure Slack
+
+1. Log in to the Slack Admin Dashboard.
+2. Navigate to the **Configure SAML Authentication** page.
+3. Enter the following values:
+    - **SAML 2.0 Endpoint (HTTP)**: copy/paste in the **SSO URL (Redirect)** URL from the provider that you created in authentik.
```

Comment:
This is where I'd usually use the placeholder `authentik.company` to give an example URL, but I suppose we don't have to do that

Reply:
Ohhhh, let's do include an example, please @BeryJu . I'm trying to learn more about these FQDN placeholders (thanks @4d62 ) but am not there yet. 

If we use it in an example here, I think I should add it back to the top of this page, under Preparation.